### DISCUSSION TITLE
Contribution to the documentation

### DISCUSSION THREAD

**Discussion by xinkev** [adrielcafe/voyager#407] [Q&A]
Hey folks! 👋
Just wanted to drop a quick question: Is it possible to contribute to the documentation website?

**Comment by DevSrSouza** [comment]
Currently no, but we are planning to move the docs to the repo, stay tuned.

**Reply by acmpo6ou** [reply]
I can see the folder `docs` is there in the `main` branch. Does that mean we can now contribute? I would like to open a PR with some improvements. There are some misspellings, etc, that I could easily fix for you if you're willing to accept my PR.
